### ISSUE TITLE
How can I update cent itslef?

### ISSUE THREAD

**Issue by Rahimianes** [xm1k3/cent#61]
Hello 
I want to know how can I update cent itself?
when I run the below command I did not see any changes into your products
go install -v github.com/xm1k3/cent@latest
would you please add a switch for update cent itself?
Thanks

**Comment by xm1k3** [comment]
Hi @Rahimianes ,
You can just add specific version on install

```
go install -v github.com/xm1k3/cent@v1.3
```

maybe i will add an automatic mode soon

**Comment by Rahimianes** [comment]
the output :

go: github.com/xm1k3/cent@v1.3: no matching versions for query "v1.3"

**Comment by xm1k3** [comment]
ok I apologize, it was probably my mistake, reading some issues also on other projects it may be due to semantic-version errors https://github.com/golang/go/issues/37529#issuecomment-592301127

so at the moment I tried to fix from v1.3 to v1.3.0
try running the install command again and give me feedback if you want, sorry again

```
go install -v github.com/xm1k3/cent@v1.3.0
```

**Comment by xm1k3** [comment]
Hi @Rahimianes 

I'm closing this issue because I haven't had any more feedback,
If there is anything else feel free to reopen and ask for what you need

Thank you
